test/Sema/enum_equatable_hashable.swift

```diff
@@ -12,9 +12,9 @@ var aHash: Int = Foo.A.hashValue
 enum Generic<T> {
   case A, B
 
-  func method() -> Int {
+  static func method() -> Int {
     if A == B { }
-    return A.hashValue
+    return Generic.A.hashValue
```

Comment:
Yep. This test is supposed to be testing synthesis of {{==}} *without* any member lookup being done, because that goes through a different path in the compiler. (Feel free to add that as a comment!)